pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -3623,10 +3623,7 @@ broken-packages:
   - clarifai
   - CLASE
   - clash
-  - clash-ghc
-  - clash-lib
   - clash-multisignal
-  - clash-prelude
```

Comment:
Could you pull out these changes and send them in a different PR against the `haskel-updates` branch?

Here's a video that describes how to mark Haskell packages as unbroken:

https://discourse.nixos.org/t/video-tutorial-how-to-fix-broken-haskell-packages-in-nix/3968

(Really the only change you need to make is to not edit the hackage-packages.nix file by hand.  It gets regenerated automatically based on this configuration-hackage2nix.yaml file.)

Reply:
done, thanks for letting me know!